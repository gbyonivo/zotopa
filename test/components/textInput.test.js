import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from '../../src/components/textInput';

describe('TextInput component', () => {
  it('TextInput renders correctly', () => {
    const actual = renderer.create(
      <TextInput
        value="play"
        label="play"
        name="play"
        placeholder="play"
        onChange={() => {}}
      />
    );
    expect(actual).toMatchSnapshot();
  });
});
