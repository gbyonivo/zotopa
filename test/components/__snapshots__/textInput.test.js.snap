// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput component TextInput renders correctly 1`] = `
<div
  className="textInput"
>
  <label
    htmlFor="play"
  />
  <input
    name="play"
    onChange={[Function]}
    placeholder="play"
    type="text"
    value="play"
  />
</div>
`;
